packages/@react-spectrum/autocomplete/intl/es-ES.json

```diff
@@ -1,6 +1,6 @@
 {
   "clear": "Borrar",
   "invalid": "(no válido)",
-  "loading": "Cargando...",
+  "loading": "Cargando…",
   "noResults": "Sin resultados"
 }
```

Comment:
ellipse updated in a couple of places, to make all strings in matching locale the same